vault/resource_database_secret_backend_connection.go

```diff
@@ -1015,6 +1027,23 @@ func getSnowflakeConnectionDetailsFromResponse(d *schema.ResourceData, prefix st
 	return []map[string]interface{}{result}
 }
 
+func getUserPassConnectionDetailsFromResponse(d *schema.ResourceData, prefix string, resp *api.Secret) []map[string]interface{} {
```

Comment:
nit: I feel like the name might be a bit off since it does more than just getting user and password. Maybe rename to `getConnectionDetailsFromResponseWithUserPass()`, so it reads more like an extension of `getConnectionDetailsFromResponse()`

Reply:
Updated in f526a7f